app/src/main/java/com/example/poste/activities/FolderViewActivity.java

```diff
@@ -72,47 +72,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         settingsBut.setOnClickListener(view -> {
             Intent intent = new Intent(FolderViewActivity.this, EditFolderActivity_v2.class);
+            intent.putExtra("folderId", PosteApplication.getSelectedFolder().getId());
```

Comment:
If we are setting PosteApplication.selectedFolder, could we just pull from that field in EditFolder activity rather than sending / retrieving extras on the intent?